doc/zstd_compression_format.md

```diff
@@ -1203,20 +1258,29 @@ _Reserved ranges :_
               - low range : 1 - 32767
               - high range : >= (2^31)
 
-__`Entropy_Tables`__ : following the same format as the tables in [compressed blocks].
+__`Entropy_Tables`__ : following the same format as the tables in compressed blocks.
+              See the relevant [FSE](#fse-table-description)
+              and [Huffman](#huffman-tree-description) sections for how to decode these tables.
               They are stored in following order :
               Huffman tables for literals, FSE table for offsets,
               FSE table for match lengths, and FSE table for literals lengths.
+              These tables populate the Repeat Stats literals mode and
+              Repeat distribution mode for sequence decoding.
+
               It's finally followed by 3 offset values, populating recent offsets (instead of using `{1,4,8}`),
```

Comment:
I think github treats paragraphs preceded by 8 spaces as code blocks unless they're part of another segment, should be fixed by removing the newline in-between.